packages/deployer/terraform/airnode/modules/function/main.tf

```diff
@@ -0,0 +1,84 @@
+resource "random_uuid" "uuid" {
+}
+
+resource "null_resource" "fetch_lambda_files" {
+  provisioner "local-exec" {
+    command = <<EOC
+rm -rf ${local.tmp_dir}
+mkdir -p "${local.tmp_source_dir}" "${local.tmp_configuration_dir}"
+cp "${var.source_file}" "${local.tmp_source_dir}"
+cp "${var.configuration_file}" "${local.tmp_configuration_dir}"
+EOC
+  }
+
+  triggers = {
+    source_file_hash        = fileexists("${local.tmp_source_dir}/${var.source_file}") ? filesha256(var.source_file) : ""
+    configuration_file_hash = fileexists("${local.tmp_configuration_dir}/${var.configuration_file}") ? filesha256(var.configuration_file) : ""
+  }
+}
+
+resource "aws_iam_role" "lambda_role" {
+  name               = "${local.name}-role"
+  assume_role_policy = data.aws_iam_policy_document.role_policy.json
+}
+
+resource "aws_cloudwatch_log_group" "cloudwatch_log_group" {
+  name              = "/aws/lambda/${local.name}"
+  retention_in_days = 14
```

Comment:
Good question. I repeated the same configuration as Serverless was using. Can definitelly be set to more. What do you suggest?

I suspect that in the future we would eventually have to allow providing some deployment options (like the retention) so even the deployment itself can be configured.

Reply:
Yeah I think API providers are going to have different requirements for how long they need to have access to logs. Would be ideal if this could be configured like you mentioned, although that doesn't need to happen now.

I would say 90 days would be better but that's just a guess. 

@bbenligiray any thoughts here?